### PR TITLE
Move Kyma Environments Cleanup job out of KCP chart

### LINKED ISSUE #2634: Disable cleanup job from control plane
**Description**

Move it out of the control-plane chart, find a new place, and then only DEV environment has it installed.

**Reason**

It is a big risk that some misconfigurations could affect production.

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Kyma Environments Cleanup job should be applied into the cluster manually.

Changes proposed:
- move Kyma Environments Cleanup job manifest out of KCP chart,
- adjust manifest to drop the Helm templating,
- add shell script to fetch the configuration values from the Kyma Environment Broker Deployment,
- add readme.

**Related issue(s)**

Resolves #2634

### CODE REVIEW

File: utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                      name: kcp-postgresql
+                - name: APP_DATABASE_SSLMODE
+                  valueFrom:
+                    secretKeyRef:
+                      key: postgresql-sslMode
+                      name: kcp-postgresql
+                - name: APP_DATABASE_SSLROOTCERT
+                  value: /secrets/cloudsql-sslrootcert/server-ca.pem
+                - name: APP_BROKER_URL
+                  value: https://$SCRIPT_BROKER_URL # set by script
```
Comment (jaroslaw-pieszka) [inline utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml:84]:
```suggestion
                  value: https://${SCRIPT_BROKER_URL} # set by script
```

File: utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                - name: gardener-kubeconfig
+                  mountPath: /gardener/kubeconfig
+                  readOnly: true
+                - name: cloudsql-instance-credentials
+                  mountPath: /secrets/cloudsql-instance-credentials
+                  readOnly: true
+            - name: cloudsql-proxy
+              image: europe-docker.pkg.dev/kyma-project/prod/tpi/cloudsql-docker/gce-proxy:v1.33.4-3cfeacc2
+              imagePullPolicy: IfNotPresent
+              command: $SCRIPT_CLOUDSQL_PROXY_COMMAND # set by script
```
Comment (jaroslaw-pieszka) [inline utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml:109]:
```suggestion
              command: ${SCRIPT_CLOUDSQL_PROXY_COMMAND} # set by script
```

File: utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                  valueFrom:
+                    secretKeyRef:
+                      key: postgresql-sslMode
+                      name: kcp-postgresql
+                - name: APP_DATABASE_SSLROOTCERT
+                  value: /secrets/cloudsql-sslrootcert/server-ca.pem
+                - name: APP_BROKER_URL
+                  value: https://$SCRIPT_BROKER_URL # set by script
+                - name: APP_BROKER_TOKEN_URL
+                  value: https://oauth2.$SCRIPT_DOMAIN/oauth2/token # set by script
```
Comment (jaroslaw-pieszka) [inline utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml:86]:
```suggestion
                  value: https://oauth2.${SCRIPT_DOMAIN}/oauth2/token # set by script
```

File: utils/kyma-environments-cleanup-job/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Kyma Environments Cleanup Job
+**Important!**  
+**The job should run only in the development environment. Make sure you are connected to the development Kubernetes cluster before applying the job.**
```
[the remark this answers is not in the document]
Comment (szwedm) [reply utils/kyma-environments-cleanup-job/README.md:3]: suggestion applied.

File: utils/kyma-environments-cleanup-job/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Kyma Environments Cleanup Job removes Kyma Environments which are older than 24h. The job is scheduled to run every day at midnight (according to the local time defined in the system).
```
[the remark this answers is not in the document]
Comment (szwedm) [reply utils/kyma-environments-cleanup-job/README.md:5]: suggestion applied.

File: utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml (hunk 1 above)
Comment (szwedm) [reply utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml:84]: suggestion applied.

File: utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml (hunk 2 above)
Comment (szwedm) [reply utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml:109]: suggestion applied.

File: utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml (hunk 3 above)
Comment (szwedm) [reply utils/kyma-environments-cleanup-job/kyma-environments-cleanup-job.yaml:86]: suggestion applied.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**